src/App.js

```diff
@@ -13,9 +13,61 @@ class App extends Component {
         <p className="App-intro">
           To get started, edit <code>src/App.js</code> and save to reload.
         </p>
+
+        <NewsListTable/>
+
+      </div>
+    );
+  }
+}
+
+class NewsListTable extends Component {
+  constructor(props) {
+    var mockNews = [
+      {
+        url: 'www.example.com',
+        user: 'tdurden',
+        title: 'Tesla Asks Suppliers for Cash Back to Help Turn a Profit',
+        timeAgo: '1 day',
+        points: '42',
+        numberOfComments: '22'
+      },
+      {
+        url: 'www.example.com',
+        user: 'jhabdas',
+        title: 'BPG Image format ',
+        timeAgo: '9 hours',
+        points: '185',
+        numberOfComments: '60'
+      }
+    ];
+    super(props);
+    this.state = {
+      news: mockNews
+    };
+  }
+
+  render() {
+    return (
+      <div>
+        <h1>News List</h1>
+        {this.state.news.map((item, index) => (
```

Comment:
It might be a good idea to create a function ```renderNewsList()```, for instance. It's always nicer to keep the render function very simple. Easier to test too :)